packages/insomnia-app/app/plugins/misc.js

```diff
@@ -230,6 +230,39 @@ function wrapStyles(theme: string, selector: string, styles: string) {
   ].join('\n');
 }
 
+export function isDarkTheme(theme: Theme) {
+  // TODO just a heuristic, should be replaced with a property on themes
+  const background = theme.theme.theme.background?.default;
+  return background && Color(background).isDark();
+}
+
+export function getColorScheme(settings: Settings): ColorScheme {
+  if (!settings.autoDetectColorScheme) {
+    return 'default';
+  }
+
+  if (window.matchMedia(`(prefers-color-scheme: light)`).matches) {
+    return 'light';
+  }
+
+  if (window.matchMedia(`(prefers-color-scheme: dark)`).matches) {
+    return 'dark';
+  }
+
+  return 'default';
+}
+
+export async function applyColorScheme(settings: Settings): Promise<IColorTheme | null> {
```

Comment:
Looks like this function doesn't return anything

```suggestion
export async function applyColorScheme(settings: Settings): Promise<void> {
```